Commands/MessageCount/pagination.go

```diff
@@ -0,0 +1,45 @@
+{{/*
+	This is the pagination code for the message leaderboard
+  
+	Recommended trigger: added reactions only
+	Trigger type: reaction
+*/}}
+{{/* dont touch */}}
+{{ $action := .Reaction.Emoji.Name }} {{/* The action being ran */}}
```

Comment:
would usually say it's an unnecessary variable creation but it is called often so you've got the argument of shortening char usage